web/packages/dmt-core/src/components/Pickers/DestinationPicker.tsx

```diff
@@ -34,6 +34,7 @@ export type DestinationPickerProps = {
 
 export const DestinationPicker = (props: DestinationPickerProps) => {
   const { onChange, formData, disabled, scope, label } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore